source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,286 @@
+
+The concordium client tool will allow you to interact with your testnet node. You find important commands that the concordium client tool provides :ref:`here<concordium_client>`.
+
+Your next task enables the concordium client tool to talk to your testnet node. There are two options to achieve this:
+
+1. Option (beginners)
+
+This option explains how to transfer the concordium client tool to your instance and execute commands from within the instance.
+
+**Advantage**: You don't have to adjust the inbound connection rules to your instance.
+
+**Disadvantage**: You have to transfer files between your local machine and your instance later in the tutorial when we deploy the smart contracts.
+
+Transfer the concordium client package from your machine via a file-sharing tool (such as `FileZilla`) to your instance.
+
+Connect to your instance and make your package executable again as we done previously already:
+
+.. code-block:: console
+
+   $chmod 777 concordium-client
+
+Let's check if everything is connected correctly by displaying a random block.
+
+.. code-block:: console
+
+   $./concordium-client block show c1ffec01a42084f4a406d6408ce435621bc5730288a447a4c8325b560e959108 --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_18.png
+   :width: 100 %
+
+.. Note::
+   Port 10001 is open by default on your testnet node to interact with it.
+
+2. Option (advanced users)
+
+This option explains how you can use the concordium client tool locally at your machine and connect remotely to your node running on the server.
+
+**Advantage**: You don't have to transfer files between your local machine and your instance later in the tutorial when we deploy the smart contracts.
+
+**Disadvantage**: You have to adjust the inbound connection rules to your instance.
+
+.. Note::
+   Port 10001 is open by default on your testnet node to interact with it. But you have to open port 10001 in your inbound connection rules for your instance as well. Please open the port only for your IP address. Please be aware of the security implication, it is assumed that you are the only person using that IP address. Please be aware that we recommend this option for advanced users.
+
+.. image:: ./images/pb_tutorial_14.png
+   :width: 100 %
+
+Let's check if everything is connected correctly by displaying a random block.
+
+.. code-block:: console
+
+   $./concordium-client block show c1ffec01a42084f4a406d6408ce435621bc5730288a447a4c8325b560e959108 --grpc-ip <IP Address of Your Instance> --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_17.png
+   :width: 100 %
+
+
+Create a mobile wallet account on testnet
+=========================================
+
+You can create an account with any of the officially provided wallets from Concordium. This tutorial focuses on the mobile wallet but feel free to explore the setup of the :ref:`desktop wallets<overview-account-desktop>` alternatively.
```

Comment:
Unfortunately, you can't use the accounts/keys created in the desktop wallet with concordium client. So the only option is to use the mobile wallet.

Reply:
removed the sentence